packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_controller_test.dart

```diff
@@ -320,57 +330,25 @@ void main() {
             LatLng(43.354469, -5.851318),
             LatLng(43.354762, -5.850824),
           ])
-        }));
-
-        controller.debugSetOverrides(
-          circles: circles,
-          markers: markers,
-          polygons: polygons,
-          polylines: polylines,
-        );
-
-        controller.init();
-
-        final Set<Circle> capturedCircles =
-            verify(circles.addCircles(captureAny)).captured[0] as Set<Circle>;
-        final Set<Marker> capturedMarkers =
-            verify(markers.addMarkers(captureAny)).captured[0] as Set<Marker>;
-        final Set<Polygon> capturedPolygons =
-            verify(polygons.addPolygons(captureAny)).captured[0]
-                as Set<Polygon>;
-        final Set<Polyline> capturedPolylines =
-            verify(polylines.addPolylines(captureAny)).captured[0]
-                as Set<Polyline>;
-
-        expect(capturedCircles.first.circleId.value, 'circle-1');
-        expect(capturedCircles.first.zIndex, 1234);
-        expect(capturedMarkers.first.markerId.value, 'marker-1');
-        expect(capturedMarkers.first.infoWindow.snippet, 'snippet for test');
-        expect(capturedMarkers.first.infoWindow.title, 'title for test');
-        expect(capturedPolygons.first.polygonId.value, 'polygon-1');
-        expect(capturedPolygons.elementAt(1).polygonId.value,
-            'polygon-2-with-holes');
-        expect(capturedPolygons.elementAt(1).holes, isNot(null));
-        expect(capturedPolylines.first.polylineId.value, 'polyline-1');
-      });
-
-      testWidgets('empty infoWindow does not create InfoWindow instance.',
```

Comment:
[Original comment by @AsturaPhoenix](https://github.com/flutter/plugins/pull/6982/files/c24d187970545ee18a52c00e8538c823a83fcf65#r1102025301):

Leaving a comment here because it wasn't obvious to me the second time I looked at it:

This test was removed because it became trivial once using Mockito verification; it essentially tests to make sure that `Marker.infoWindow` defaults to `InfoWindow.noText`. On the other hand, the argument could be made to keep it anyway (e.g. to construct the verification with an explicit `infoWindow: InfoWindow.noText`); let me know if you'd prefer that.